MonoGame.Framework/Audio/OpenALSoundController.cs

```diff
@@ -291,8 +290,25 @@ public static OpenALSoundController GetInstance
         {
 			get
             {
-				if (_instance == null)
-					_instance = new OpenALSoundController();
+                if (_instance == null)
+                {
+                    try
```

Comment:
This `GetInstance` property doesn't really make sense to me, at least with this naming. Should be a method IMO.
Also would make more sense to be able to call an initialize function instead of this to initialize OpenAL. Can you extract the code here into an `EnsureInitialized` function:

```
public static void EnsureInitialized()
{
    if (_instance ~= null)
        return;
    try
    {
    ...
}

public static OpenALSoundController GetInstance()
{
    EnsureInitialized();
   return _instance;
}
```

Reply:
Yeah, this GetInstance has been there for a long time and has been used to initialize stuff at several places...

I think we should turn it into a property and make it throw if it is called before the initialization, so that the code the more readable with no hidden initialization due to a property accessor.

Reply:
> I think we should turn it into a property and make it throw if it is called before the initialization, so that the code is more readable with no hidden initialization due to a property accessor.

 Even better